lib/features/sks-menu/data/models/sks_menu_response.dart

```diff
@@ -16,3 +16,16 @@ class SksMenuResponse with _$SksMenuResponse {
   factory SksMenuResponse.fromJson(Map<String, dynamic> json) =>
```

Comment:
Do we need `SksMenuResponse `any more?

Reply:
it's just an easy way of parsing the original response, so imo pretty much yes